Program.cs

```diff
@@ -299,14 +299,15 @@ async Task MessageUpdated(DiscordClient client, MessageUpdateEventArgs e)
             commands = discord.UseCommandsNext(new CommandsNextConfiguration
             {
                 StringPrefixes = cfgjson.Core.Prefixes
-            }); ;
+            });
 
             commands.RegisterCommands<Warnings>();
             commands.RegisterCommands<MuteCmds>();
             commands.RegisterCommands<UserRoleCmds>();
             commands.RegisterCommands<ModCmds>();
             commands.RegisterCommands<Lockdown>();
             commands.RegisterCommands<Bans>();
+            commands.RegisterCommands<Reports>();

```

Comment:
Should the !report command added as a slash command for more visibility?

Reply:
I think it would be a good idea to make users aware about the command (only /report and /report_u maybe?)

Reply:
I think so, just hide the mod commands like agree with complaint or ignore.